how-tos/homework-submission.md

```diff
@@ -0,0 +1,35 @@
+#Instructions for Submitting Homework
+
+## Submitting Dawn/Dusk Exercises
+
+Please submit your solutions to **dawn/dusk exercises** using your homework repo.
+
+- Copy/paste your solutions into your homework repo (e.g. john-doe/week-01/day-01/solutions.md)
+- Follow the instructions below for *Creating Pull Requests*.
+
+## Submitting Lab Work
+Unlike dawn/dusk exercises, *labs* are projects you started in class that you had to fork and clone. These require a slightly different submission flow:
+
+- When you begin a lab, you must remember to first `fork` the lab! (Just click the `fork` button).
+- Next, `git clone` your fork of the lab so that you can work on it locally (on your computer).
+- Congrats! You're ready to tackle the lab!
+
+
+### Creating Pull Requests
```

Comment:
Should this section be moved under Submitting Dawn/Dusk Exercises?  (or made into an h2)